apps/qubit/modules/digitalobject/actions/metadataComponent.class.php

```diff
@@ -40,5 +40,17 @@ public function execute($request)
     {
       $this->denyFileNameByPremis = true;
     }
+
+    // Provide Google Maps API key to template
+    $keyProperty = QubitSetting::getByName('google_maps_api_key');
```

Comment:
If you use the sfConfig store instead you'll save a hit to the database,  settings are populated in `sfConfig` from our cache backend.